Doc/library/typing.rst

```diff
@@ -2604,10 +2604,12 @@ Functions and decorators
 
 .. function:: reveal_type(obj, /)
 
-   Reveal the inferred static type of an expression.
+   Ask a static type checker to reveal the statically inferred type of an
+   expression.
 
    When a static type checker encounters a call to this function,
-   it emits a diagnostic with the type of the argument. For example::
+   it emits a diagnostic with the inferred static type of the argument.
```

Comment:
Same here. The addition of "inferred" is good, but I don't think we need "static" twice in the same sentence:

```suggestion
   it emits a diagnostic with the inferred type of the argument.
```